scipy/stats/tests/test_distributions.py

```diff
@@ -3694,6 +3694,40 @@ def test_skewcauchy_R(self):
         assert_allclose(stats.skewcauchy.ppf(cdf, a), x)
 
 
+class TestJFSkewT:
+    def test_stats(self):
+        actual = stats.jf_skew_t(8, 4).stats('mvsk')
+        expected = [
+            1.4247801402401363,
+            1.684287266263012,
+            0.7983625393780869,
+            2.3345439210504404,
```

Comment:
How were the expected values computed?

Reply:
These were computed by me in a Jupyter notebook. I could also have used some parameters which give simpler stats, e.g. `a=b=3` would give stats `[0, 1.5, 0, 3]` (which is easy to verify because a=b=3 gives a t distribution with 6 degrees of freedom).

It seemed like a good idea to have this test mainly to flag any issues that changes to the code might cause.

I'll take others' lead on whether it's needed or not though.

Reply:
I've removed this test now.